src/components/ReviewExtraInfo.jsx

```diff
@@ -2,29 +2,38 @@ import axios from "axios";
 import { useEffect } from "react";
 import { useState } from "react";
 import { useParams } from "react-router-dom"
+import Comments from "./Comments";
 
 const ReviewExtraInfo = () => {
     const {review_id} = useParams();
 
     const [review, setReview] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
 
     useEffect(() => {
         axios.get(`https://nc-games-no2.onrender.com/api/reviews/${review_id}`)
         .then(({data}) => {
             setReview(data)
+            setIsLoading(false)
         })
-    }, [review_id]);
+    }, []);
+
+    if (isLoading) {
+        return <p>Loading review...</p>
+      };
 
     return (
-        <>
+        <div className="review-info">
         <h1>{review.title}</h1>
         <h3>Written by: {review.owner}</h3>
-        <p>Created at: {review.created_at}</p>
+        <time>Created at: {review.created_at}</time>
         <p>Category: {review.category}</p>
         <p>Votes: {review.votes}</p>
         <img src={review.review_img_url} alt={review.title}></img>
         <p>{review.review_body}</p>
-        </>
+        <Comments review_id={review_id}/>
+        </div>
+        
     )
 
 
```

Comment:
This component looks really good, however I'd consider having the link to it be the entire card rather than just the title to make it easier and more intuitive for users to click on if for instance they clicked the image rather than the title.